frameworks/C++/includeos-mana/benchmark_config.json

```diff
@@ -0,0 +1,24 @@
+{
+  "framework": "includeos-mana",
+  "tests": [{
+    "default": {
+      "setup_file": "setup",
+      "plaintext_url": "/plaintext",
+      "port": 8080,
+      "approach": "Stripped",
+      "classification": "Platform",
+      "database": "None",
+      "framework": "Mana",
```

Comment:
This should be `None` if the classification is `Platform` (or if there is a different framework used, list that here)